Documentation/en/relayserver.md

```diff
@@ -15,6 +15,13 @@ The goal of this list is to highlight companies who pay back to this open source
 [<img width="120px" src="./assets/logo_sponsor_cmi.svg" />](https://www.cmiag.ch/)
 
 # Version history
+
+## Version 2.0.1
+
+- Buf fixes
```

Comment:
This is a typo.
```suggestion
- Bugfixes
```

Reply:
That was spelled `Bug fixes` in the 2.0.0 release notes, so we should stick to that. Fixed the typo, though ;)